presto-spi/src/main/java/com/facebook/presto/spi/connector/ConnectorMetadata.java

```diff
@@ -238,6 +238,12 @@ default void finishCreateTable(ConnectorSession session, ConnectorOutputTableHan
     }
 
     /**
+     * Start a SELECT/UPDATE/INSERT/DELETE query. This notification is triggered after the planning phase completes.
```

Comment:
I made a distinction because I introduced first part of beginQuery() - for select, insert etc. (in 46fefccf6edf927b0898b5fc0c8733cfc5b8e556) then I extended it for DDLs (after a few commits as prerequisites) in 32b35c3d454aa3a9becd32aee1c0a725c9a7c947.
So in first commit I refer to queries I cover (select, update, insert, delete) and in latter I generalize it and change comment to simple "Start a query".
I believe it's more informative that way because whichever commit you checkout, comment is consistent with it. But if you insist I can change this comment here.